.rubocop.yml

```diff
@@ -34,3 +34,6 @@ Layout/EmptyLineBetweenDefs:
 Metrics/LineLength:
   Max: 150
 
+# TODO: Need to fix below one
```

Comment:
Do we want to address this todo now?  I'm a bit nervous about adding tech debt to the codebase, but I'm ok with not extending this pr further.  Would you mind explaining what you'd like to do to address, and if we won't be taking care of it, let's put more of an explanation in the doc string for the future maintainer.

Reply:
@daryanypl  I will try to fix it in next PR. If  I am unable to fix  above issue than i will add some documentaton.

Reply:
@daryanypl Created user story for test cases. All test cases will address one time.